tests/test_servefile.py

```diff
@@ -117,6 +119,22 @@ def check_download(expected_data=None, path='/', fname=None, status_code=200, **
     return r  # for additional tests
 
 
+def _retry_while(exception, function, timeout=2):
+    now = datetime.datetime.now
+
+    def wrapped(*args, **kvargs):
```

Comment:
```suggestion
    def wrapped(*args, **kwargs):
```

Reply:
Happens to me all the time: It's "key value" to me and rather than "keywords" for some reason. I'll fix it in a second